modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -435,6 +415,44 @@ FROM (
 
 You can now insert the transformed record back into the main Iceberg table. Redpanda recommends employing a strategy for exactly-once processing to avoid duplicates when reprocessing records.
 
+== Performance considerations
+
+When you enable Iceberg for any substantial workload, you may see most of your cluster's CPU utilization increase as a result of translation of topic data to the Iceberg format. If this process overloads the brokers, Redpanda automatically applies backpressure to clients to prevent runaway lag of the Iceberg table behind the topic log.
```

Comment:
```suggestion
When you enable Iceberg for a substantial workload and start translating topic data to Iceberg format, you may see your cluster's CPU utilization increase. If this process overloads the brokers, Redpanda automatically applies backpressure to clients to prevent the Iceberg table from falling too far behind the topic log.
```
Just a suggestion if you find this easier to process.

Reply:
"When you enable Iceberg for a substantial workload and start translating topic data to Iceberg format, this will cause increased CPU utilization due to the additional workload. If this workload overloads the brokers, causing the iceberg table's lag to rise above the configured target lag, Redpanda automatically applies backpressure to Producer clients to prevent the Iceberg table from lagging further, sacrificing the ingress throughput to Redpanda in order to ensure iceberg tables are able to keep up with the volume of incoming data.  You may need to increase the size of your Redpanda cluster to accommodate this additional workload. Please work with your Redpanda Customer Success team for guidance related to sizing and scaling up your cluster"